Userland/Applications/FontEditor/FontEditor.cpp

```diff
@@ -415,9 +418,20 @@ FontEditorWidget::FontEditorWidget(const String& path, RefPtr<Gfx::BitmapFont>&&
     };
 
     m_weight_combobox->on_change = [this](auto&, auto&) {
-        m_edited_font->set_weight(GUI::name_to_weight(m_weight_combobox->text()));
+        m_edited_font->set_weight(Gfx::name_to_weight(m_weight_combobox->text()));
```

Comment:
What happened to the ::GUI version of this function and its friends?

Reply:
These functions come from the now deleted FontPickerWeightModel.h. FontPicker no longer needs that model, but its associated mapping is still useful and used by BitmapFonts, so it was moved to LibGfx in FontStyleMapping.h. Made this more clear in the commit.